src/meltano/core/tracking/iglu-client-embedded/schemas/com.meltano/plugins_context/jsonschema/1-0-0

```diff
@@ -26,40 +26,46 @@
                     "category": {
                         "description": "",
                         "type": "string",
-                        "enum": [
-                            "extractor",
-                            "loader",
-                            "transformer"
-                        ]
+                        "maxLength": 64
                     },
                     "name_hash": {
                         "description": "64 character hash of the plugin name",
                         "type": "string",
                         "maxLength": 64
                     },
-                    "parent_name_hash": {
-                        "description": "64 character hash of the parent plugin name",
+                    "namespace_hash": {
+                        "description": "64 character hash of the plugin namespace",
+                        "type": "string",
+                        "maxLength": 64
```

Comment:
```suggestion
                        "minLength": 64,
                        "maxLength": 64
```

Reply:
@WillDaSilva we'll have to update these in a follow up, since it will require release a new version of the schema since its already live and in "production" on Snowcat cloud.